doc/source/how-to/index.rst

```diff
@@ -6,8 +6,103 @@ effective and efficient Python libraries to interface with Ansys products and
 services. Topics also demonstrate how apps and complex services expose
 functionalities such as logging, data transfer, and app APIs.
 
+.. grid:: 3
+
+    .. grid-item-card:: :fas:`fa-regular fa-screwdriver-wrench` Environment setup
+      :link: setting-up
+      :link-type: doc
+      :padding: 2 2 2 2
+
+      How to set up a development environment.
+
+    .. grid-item-card:: :fas:`fa-solid fa-code-compare` Versions
+      :link: supporting-python-versions
+      :link-type: doc
+      :padding: 2 2 2 2
+
+      Supporting Python versions.
+
+    .. grid-item-card:: :fas:`fa-solid fa-people-group` Contributing
+      :link: contributing
+      :link-type: doc
+      :padding: 2 2 2 2
+
+      Learn how to contribute to PyAnsys.
```

Comment:
```suggestion
      How to contribute to PyAnsys.
```
Changed so that this how-to description matched all the other "how to" descriptions.